### DISCUSSION TITLE
Form submission by username versus by code

### DISCUSSION THREAD

**Discussion by saseehav** [precision-sustainable-ag/tech-dashboard#755] [Ideas]
some forms will be submitted by MD for example (dry biomass bags that need to be weighed and ground) but they will be intended for other states. Can they be sorted by code?

**Comment by brianwdavis** [comment]
I think that MD becomes responsible for making sure those forms are input and parsed correctly, if they're the one who submitted it. Just because state X owns the bags, doesn't mean they know what to do with the form submission. Once it's parsed into the DB, that data will show up for state X though.
